packages/labs/testing/README.md

```diff
@@ -0,0 +1,95 @@
+# @lit-labs/testing
+
+A package containing utilities for testing Lit.
+
+## Status
+
+`@lit-labs/testing` is part of the [Lit
+Labs](https://lit.dev/docs/libraries/labs/) set of packages - it is published in
+order to get feedback on the design and not ready for production. Breaking
+changes are likely to happen frequently.
+
+## Usage
+
+### Web Test Runner Plugin for Lit SSR
+
+Add the plugin to your config file.
+
+```js
+// web-test-runner.config.js
+import {litSsrPlugin} from '@lit-labs/testing';
+
+export default {
+  plugins: [litSsrPlugin()],
+};
+```
+
+This plugin registers the `lit-ssr-render` server command utilized by the SSR
+fixture functions.
+
+### Fixtures
+
+The package exports functions that will generate fixtures by passing the
+provided lit-html templates to the dev server to be rendered server-side, and
+added to the browser document. The Web Test Runner plugin must be added to the
+config file for this to work.
+
+#### `ssrFixture`
+
+Example
+
+```js
+// my-element.test.js
+import {ssrFixture} from '@lit-labs/testing/fixtures.js';
+import {html} from 'lit';
+import {assert} from '@esm-bundle/chai';
+
+suite('my-element', () => {
+  test('is rendered server-side', async () => {
+    const el = await ssrFixture(html`<my-element></my-element>`, {
+      modules: ['./my-element.js'], // path to component definition relative to project root
+      hydrate: false, // whether to hydrate the component after loading to document (default: true)
+    });
+    assert.equal(el.shadowRoot.querySelector('p').textContent, 'Hello, World!');
+  });
+});
+```
+
+#### `ssrNonHydratedFixture` and `ssrHydratedFixture`
```

Comment:
I forget what we concluded, but I was also expecting a CSR version with the same signature that used the client-side `render()`?

Reply:
Yes, we did want that. It's on the todo list.